MicroProfile-Rest-Client/tck-runner/pom.xml

```diff
@@ -138,7 +139,7 @@
         <dependency>
             <groupId>org.eclipse.microprofile.rest.client</groupId>
             <artifactId>microprofile-rest-client-api</artifactId>
-            <version>${microprofile.rest.client.tck.version}</version>
+            <version>${microprofile.rest.client.api.version}</version>
```

Comment:
I'm failing to see how this would fix anything? This property isn't modified by any profile so will always be set to 1.2.1.payara-p1?

Reply:
Indeed that is the point, the change of the tck.version property was causing an old version of the api to be used against 4 (causing the failure) so the api should always be 1.2.1.payara-p1 or whatever version may be consistently applicable in future - just added a new property for the sakes of good practice in my eyes